tests/errors/test_pow3.py

```diff
@@ -0,0 +1,5 @@
+from ltypes import c32
+def f():
+    x: c32
+    x = 4 + 6j
+    print(pow(x,3))
```

Comment:
```suggestion
    print(pow(x,3))

```

Reply:
This should work. Its totally fine to raise complex numbers to some power. See a CPython example  below,

```python
Python 3.8.9 (default, Apr 13 2022, 08:48:06) 
[Clang 13.1.6 (clang-1316.0.21.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> (4 + 6j)**2
(-20+48j)
```